mlrun/config.py

```diff
@@ -538,7 +538,7 @@
     "workflows": {
         "default_workflow_runner_name": "workflow-runner-{}",
         # Default timeout seconds for retrieving workflow id after execution:
-        "timeouts": {"local": 120, "kfp": 30},
+        "timeouts": {"local": 120, "kfp": 30, "remote": 30},
```

Comment:
Where this one is being used ?

Reply:
Right here: https://github.com/mlrun/mlrun/blob/549b6678b5597bf2875ed29f59a5e33e278d7af7/mlrun/projects/pipelines.py#L810

It seems to me that all engines require a default timeout because of that.

Let me know if you believe this should be addressed on another PR or if you wanna discuss this separately. I added it here, because I think this will be the next error on the test pipelines.

Here an example of the error I found:
```
> 2023-10-07 01:24:30,147 [info] Submitting remote workflow: {'workflow_engine': 'remote', 'schedule': None, 'project_name': 'load-project-0'}
> 2023-10-07 01:24:31,452 [error] Traceback (most recent call last):
  File "/Users/laury/dev/iguazio/misc_workspace/mlrun/mlrun/projects/pipelines.py", line 810, in run
    getattr(mlrun.mlconf.workflows.timeouts, inner_engine.engine),
  File "/Users/laury/dev/iguazio/misc_workspace/mlrun/mlrun/config.py", line 626, in __getattr__
    raise AttributeError(attr)
AttributeError: remote

[error] :x: Workflow main run failed!, error: remote
Traceback (most recent call last):
  File "/Users/laury/dev/iguazio/misc_workspace/mlrun/mlrun/projects/pipelines.py", line 810, in run
    getattr(mlrun.mlconf.workflows.timeouts, inner_engine.engine),
  File "/Users/laury/dev/iguazio/misc_workspace/mlrun/mlrun/config.py", line 626, in __getattr__
    raise AttributeError(attr)
AttributeError: remote
```